src/commands/new/glee.ts

```diff
@@ -1,55 +1,225 @@
-import {Flags} from '@oclif/core';
+import { Flags } from '@oclif/core';
 import { promises as fPromises } from 'fs';
 import Command from '../../base';
 import { resolve, join } from 'path';
 import fs from 'fs-extra';
-
+import { Specification, load } from '../../models/SpecificationFile';
+import path from 'path';
+import yaml from 'js-yaml';
+import { prompt } from 'inquirer';
 export default class NewGlee extends Command {
   static description = 'Creates a new Glee project';
 
   protected commandName = 'glee';
 
   static flags = {
     help: Flags.help({ char: 'h' }),
-    name: Flags.string({ char: 'n', description: 'name of the project', default: 'project' }),
-    template: Flags.string({ char: 't', description: 'name of the template', default: 'default' }),
+    name: Flags.string({
```

Comment:
unable to regenerate the glee project as it is missing the `--force-write` flag. 

<img width="1076" alt="Screenshot 2024-01-23 at 7 38 56 PM" src="https://github.com/asyncapi/cli/assets/41781438/806c9a4c-d349-40d9-b2e2-d44fdeaaf5c7">
<img width="437" alt="Screenshot 2024-01-23 at 7 39 03 PM" src="https://github.com/asyncapi/cli/assets/41781438/cbea10d2-6127-4f01-8fe8-cf9905d7c5b8">

Reply:
it is a requirement by the generator. you can't generate on a git repository with upstaged changes.

Reply:
Yeah, but can't we pass in the flag from our CLI, since `generate fromTemplate` does that?

Reply:
@Souvikns  I think it is best to expose `--force-write` in `asyncapi new glee` command. and leave it to the user if they want to force write or not. 🤷 

@ayushnau can you add `--force-write` flag to `asyncapi new glee` command and then pass it on to the generator if user passes it? 🤔

Reply:
ok sure will do that.

Reply:
@KhudaDad414 Added the requied changes. please review.

Reply:
and the its /rtm

Reply:
/rtm

Reply:
@KhudaDad414 i have Added force-write flag. please review.

Reply:
@AyushNautiyalDeveloper 
`--force-write` flag addition looks good to me 👍